presto-hive/src/main/java/io/prestosql/plugin/hive/security/SqlStandardAccessControl.java

```diff
@@ -95,6 +95,7 @@
     public static final String ADMIN_ROLE_NAME = "admin";
     private static final String INFORMATION_SCHEMA_NAME = "information_schema";
     private static final SchemaTableName ROLES = new SchemaTableName(INFORMATION_SCHEMA_NAME, "roles");
+    private static final SchemaTableName ROLE_GRANTS = new SchemaTableName(INFORMATION_SCHEMA_NAME, "role_authorization_descriptors");
```

Comment:
rename this variable to match the value it holds

Reply:
Done